docs/r/cdap_profile.md

```diff
@@ -0,0 +1,46 @@
+# cdap_profile
```

Comment:
Added a note at the top that these documents are auto-generated and should not be modified manually?